docs/userGuide/syntax/textStyles.md

```diff
@@ -26,10 +26,14 @@ Syntax added by MarkBind:
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">markdown</variable>
 <variable name="code">
-****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~
+****Super Bold****, !!Underline!!, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~, #r#Coloured Text##
```

Comment:
Quick review: doesn't seem to work here
![image](https://user-images.githubusercontent.com/41845017/183624633-106bf360-5093-41eb-8b77-9625246daee9.png)

Reply:
Also, quick check, does it work in a code block? e.g. `some code then #r#text## `(not supposed to)

Reply:
> Also, quick check, does it work in a code block? e.g. `some code then #r#text## `(not supposed to)

Ah it still changes to the span, let me go and fix it!

Reply:
I think coloring should be explained as a separate feature, as the syntax is different, and not part of markdown.  `Tip` boxes are not suitable for giving regular instructions anyway (they are meant to be to be optional info).